php/class-terminus.php

```diff
@@ -162,15 +162,37 @@ static function prompt($message = '', $params = array(), $default=null) {
   }
 
   /**
-   * Prompt the user for input
+   * Gets input from STDIN silently
+   * By: Troels Knak-Nielsen
+   * From: http://www.sitepoint.com/interactive-cli-password-prompt-in-php/
    *
-   * @param string $message
+   * @param [string] $message Message to give at prompt
+   * @param [array]  $params  Params given to command
+   * @param [string] $default Default value
+   * @return [string] $message
    */
-  static function promptSecret($message = '', $params = array(), $default=null) {
-    exec("stty -echo");
-    $response = Terminus::prompt($message, $params);
-    exec("stty echo");
-    Terminus::line();
+  static function promptSecret($message = '', $params = array(), $default = null) {
+    if (preg_match('/^win/i', PHP_OS)) {
```

Comment:
Ok, I guess we use it [one other place](https://github.com/pantheon-systems/cli/blob/4fb53c90f6fb346e5c33f8a6c6e3dcf1de1a551d/php/commands/help.php#L97). Maybe we should use the util function.